### PR TITLE
UP-159: Add CI configuration for building and deploying snapshots upon each commit

### PR DESCRIPTION
**Pull request by rubailly**
- Updated `distributionManagement` section of pom.xml

- Added `deploy-snapshot`, `perform-release` and `verify-prs` workflows

- Updated `README.md` file

### CODE REVIEW

File: .github/workflows/perform-release.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+# This needs to be executed by issuing an authenticated REST Post operation, to prevent accidental or unauthorized releases.
+# This should involve a command such as the following:
+#
+#    export GITHUB_TOKEN=your-personal-access-token
+#    curl  -H "Authorization: token $GITHUB_TOKEN" \
+#      -H "Accept: application/vnd.github.everest-preview+json"  \
+#      -H "Content-Type: application/json" \
+#      --request POST \
+#      --data '{ "event_type": "perform-release", "client_payload": { "ref": "master" } }' \
+#      https://api.github.com/repos/Rwanda-EMR/openmrs-distro-rwandaemr/dispatches
```
Comment (mseaton) [inline .github/workflows/perform-release.yml]: This comment line needs to be adjusted to https://api.github.com/repos/Rwanda-EMR/openmrs-module-pharmacymanagement/dispatches

**Review by mseaton — APPROVED** [review]
LGTM

File: .github/workflows/perform-release.yml (hunk 1 above)
Comment (rubailly) [reply .github/workflows/perform-release.yml]: Done.

Comment (rubailly) [reply .github/workflows/perform-release.yml]: Thanks @mseaton . This is merged.

**Pull request merged**